pupil-spa/src/app/submission-pending/submission-pending.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { environment } from '../../environments/environment';
+import { SubmissionService } from '../services/submission/submission.service';
+import { AuditService } from '../services/audit/audit.service';
+import { Router } from '@angular/router';
+import { CheckSubmissionApiCalled, CheckSubmissionAPICallSucceeded } from '../services/audit/auditEntry';
+
+@Component({
+  selector: 'app-submission-pending',
+  templateUrl: './submission-pending.component.html',
+  styleUrls: ['./submission-pending.component.scss']
+})
+export class SubmissionPendingComponent implements OnInit {
+
+  @Output()
+  clickEvent: EventEmitter<any> = new EventEmitter();
+
+  constructor(private submissionService: SubmissionService,
+              private auditService: AuditService,
+              private router: Router) {
+  }
+
+  async ngOnInit() {
```

Comment:
One of the benefits of being inside the Check Component is that it handles reloads well.   What happens here if a reload occurs?   You would need to listen to the response from the server (hopefully saying it had already been submitted).  @GuyHarwood - thoughts?

Reply:
I could add something like this to block it with a in-browser pop up but that doesn't solve the problem I guess.
    `window.addEventListener('beforeunload',  (e) => {
      const confirmationMessage = '';
      e.returnValue = confirmationMessage;     // Gecko, Trident, Chrome 34+
      return confirmationMessage;              // Gecko, WebKit, Chrome <34
    });`
Waiting on thoughts from @GuyHarwood